src/components/Header/Nav.tsx

```diff
@@ -1,21 +1,24 @@
 import { navLinks } from "@/constants";
 import { isActiveRoute, isExternalLink } from "@/helpers";
 import NextLink from "next/link";
-import { usePathname } from "next/navigation";
+import { usePathname, useSearchParams } from "next/navigation";
 import ExternalLink from "public/assets/icons/external-link.svg";
 import type { CSSProperties } from "react";
 import styled from "styled-components";
 
 export function Nav() {
   const pathname = usePathname()!;
+  const searchParams = useSearchParams();
 
   return (
     <Wrapper>
       <NavItems>
         {navLinks.map(({ label, href }) => (
           <NavItem key={href}>
             <Link
-              href={href}
+              href={`${href}${
```

Comment:
pretty annoying that you have to do this. the new router treats search params as just another part of the url, which is true I guess. but that means that you have to explicitly keep the search params around when you navigate to a different url